analysis/config/config.go

```diff
@@ -518,6 +543,20 @@ func (c Config) MatchCoverageFilter(filename string) bool {
 	}
 }
 
+// IsPathSensitiveFunc returns true if funcName matches any regex in c.Options.PathSensitiveFuncs.
+func (c Config) IsPathSensitiveFunc(funcName string) bool {
```

Comment:
Given the new annotations support, it might makes sense to unify the config so you can specify function-level attributes (such as path sensitivity, allowlisting, escape analysis type, etc via either an source annotation or a config regex. This would support both code we control as well as "annotating" system code.